### PR TITLE
Chain Registry update for Router Chain

### PR DESCRIPTION
**Pull request by Iamshashvat**
Chain Registry update for Router Chain

### CODE REVIEW

File: routerchain/chain.json [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+      {
+        "image_sync": {
+          "chain_name": "router chain",
+          "base_denom": "route"
+        },
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/routerchain/images/pngRouter.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/routerchain/images/routerLogo.svg",
+        "theme": {
+          "primary_color_hex": "#cb0766",
+          "circle": true
```
Comment (JeremyParish69) [inline routerchain/chain.json:162]: These are logomarks, but not cirular

File: routerchain/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "routerchain",
+    "assets": [
+      {
+        "description": "The native token of Router Chain",
+        "extended_description": "Router Chain is a PoS L1 that allows stateful bridging–enabling truly omnichain dapps that can abstract chains from the users and aggregate liquidity across multiple chains.",
+        "denom_units": [
```
Comment (JeremyParish69) [inline routerchain/assetlist.json:8]: The asset requires a base unit with 0 exponent.

File: routerchain/chain.json [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+      {
+        "kind": "Router Explorer",
+        "url": "https://router.explorers.guru/",
+        "tx_page": "https://router.explorers.guru/transaction/${txHash}"
+      }
+    ],
+    "images": [
+      {
+        "image_sync": {
+          "chain_name": "router chain",
```
Comment (JeremyParish69) [inline routerchain/chain.json:155]: router chain would be one word here, so "routerchain"

File: routerchain/chain.json [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+          "base_denom": "route"
+        },
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/routerchain/images/pngRouter.png",
```
Comment (JeremyParish69) [inline routerchain/chain.json:158]: For visually identical images, it's best to use the same file name

File: routerchain/chain.json [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+        {
+          "denom": "route"
+        }
+      ]
+    },
+    "codebase": {
+      "versions": [
+        {
+          "name": "v2.1.1",
+          "recommended_version": "v2.1.1",
```
Comment (JeremyParish69) [inline routerchain/chain.json:38]: Please copy recommended_version and compatible_versions outside the versions array (but within codebase)

File: routerchain/chain.json [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+          "compatible_versions": [
+            "v2.1.1"
+          ]
+        }
+      ],
+      "binaries": {
```
Comment (JeremyParish69) [inline routerchain/chain.json:44]: please copy from binaries to cosmwasm_enabled inside the versions array as well.

File: routerchain/assetlist.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+          {
+            "denom": "route",
+            "exponent": 18
+          }
+        ],
+        "base": "route",
+        "name": "Router Chain",
+        "display": "route",
+        "symbol": "ROUTE",
+        "coingecko_id": "router-protocol",
```
Comment (JeremyParish69) [inline routerchain/assetlist.json:18]: As Per https://api.coingecko.com/api/v3/coins/list, the coingecko-id is just "route"

File: routerchain/chain.json (hunk 1 above)
Comment (Iamshashvat) [reply routerchain/chain.json:162]: I am removing circle: true

File: routerchain/chain.json [hunk 8]
```diff
@@ -0,0 +1,166 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "router chain",
+    "chain_type": "cosmos",
+    "chain_id": "9600",
+    "pretty_name": "Router Protocol",
+    "status": "live",
+    "network_type": "mainnet",
+    "website": "https://www.routerprotocol.com/",
+    "bech32_prefix": "router",
```
[the remark this answers is not in the document]
Comment (Iamshashvat) [reply routerchain/chain.json:10]: https://github.com/satoshilabs/slips/pull/1766 PR raised

File: routerchain/assetlist.json (hunk 2 above)
Comment (Iamshashvat) [reply routerchain/assetlist.json:8]: currently we have not set to any.

File: routerchain/chain.json (hunk 8 above)
[the remark this answers is not in the document]
Comment (JeremyParish69) [reply routerchain/chain.json]: Thanks. The PR raised has value "route", which the value here says "router". Which is correct?

File: routerchain/assetlist.json (hunk 2 above)
Comment (JeremyParish69) [reply routerchain/assetlist.json:8]: Every asset must have a base denomination that represents indivisible units. If the on-chain unit is "route", then I'd recommend using "route" as exponent 0 (the base denom), and  "ROUTE" as exponent 18 (the display denom).

File: routerchain/chain.json (hunk 8 above)
[the remark this answers is not in the document]
Comment (Iamshashvat) [reply routerchain/chain.json]: Thanks. I have fixed the PR

File: routerchain/assetlist.json (hunk 2 above)
Comment (Iamshashvat) [reply routerchain/assetlist.json:8]: To have this, we have to configure same on router chain also.

Comment (Iamshashvat) [reply routerchain/assetlist.json:8]: Hey @JeremyParish69, we have added the denom.

File: routerchain/assetlist.json [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+            "exponent": 0
+          },
+          {
+            "denom": "ROUTE",
+            "exponent": 18
+          }
+        ],
+        "base": "route",
+        "name": "Router Chain",
+        "display": "route",
```
Comment (JeremyParish69) [inline routerchain/assetlist.json:20]: this display denom should now correspond to the unit to be displayed, which I presume is "ROUTE"

**Pull request merged**